.circleci/config.yml

```
@@ -304,4 +316,11 @@ workflows:
           requires:
             - "Test: lint"
             - "Test: flow"
-            - "Build: Android release apk"
\ No newline at end of file
+            - "Build: Android release apk"
+      - "Publish: semantic release":
+          requires:
+            - "Test: Android e2e"
+            - "Test: iOS e2e"
+          filters:
+            branches:
+              only: master
```

New line at end of file

```suggestion
              only: master

```